src/components/DxpUserProfile.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="user-profile">
+    <ion-card>
+      <ion-item lines="full">
+        <ion-avatar slot="start" v-if="userProfile?.partyImageUrl">
+          <DxpImage :src="userProfile.partyImageUrl"/>
+        </ion-avatar>
+        <!-- ion-no-padding to remove extra side/horizontal padding as additional padding 
+        is added on sides from ion-item and ion-padding-vertical to compensate the removed
+        vertical padding -->
+        <ion-card-header class="ion-no-padding ion-padding-vertical">
+          <ion-card-subtitle>{{ userProfile.userLoginId }}</ion-card-subtitle>
+          <ion-card-title>{{ userProfile.partyName }}</ion-card-title>
+        </ion-card-header>
+      </ion-item>
+      <ion-button color="danger" @click="logout()">{{ profileButtonsText.logoutText }}</ion-button>
```

Comment:
Here we can take two different props as `logoutLabel` and `goToLabel` instead of defining them as a single object.